src/models/MatrixDataManager.ts

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
This file isn't actually used by the new update, i thought that it would be needed (which is why it was created), because we now resolve aliases before storing them.